README.md

```diff
@@ -10,22 +10,28 @@ The dependency list is short: numpy and Cython (>= 0.21.1).
 Usage
```

Comment:
Please add some context and details to this README:
* What is the main functionality in this repo?  Why should I care about it?
* Once we have learned a vocab how do we use it?  What are some interesting things we can use it to do?